kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -0,0 +1,2109 @@
+        for device, spec in desired.ip_addresses.items():
+            _validate_ovs_name(device)
+            _validate_ip_address(spec.address)
+            r, o, e = bash.bash_roe('ip addr flush dev %s' % device)
+            if r != 0:
+                raise Exception('failed to flush IP addresses on %s: %s'
+                                % (device, e))
+            r, o, e = bash.bash_roe('ip addr add %s dev %s' % (spec.address, device))
+            if r != 0:
+                raise Exception('failed to assign IP %s to %s: %s'
+                                % (spec.address, device, e))
+            r, o, e = bash.bash_roe('ip link set %s up' % device)
```

Ruff (0.15.12)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)